packages/web-api/src/controllers/scan-report-controller.ts

```diff
@@ -0,0 +1,50 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+import { GuidGenerator, ServiceConfiguration } from 'common';
+import { inject, injectable } from 'inversify';
+import { Logger } from 'logger';
+import { ApiController, PageScanRunReportService } from 'service-library';
+
+@injectable()
+export class ScanReportController extends ApiController {
+    public readonly apiVersion = '1.0';
+    public readonly apiName = 'web-api-get-report';
+
+    public constructor(
+        @inject(PageScanRunReportService) private readonly pageScanRunReportService: PageScanRunReportService,
+        @inject(GuidGenerator) protected readonly guidGenerator: GuidGenerator,
+        @inject(ServiceConfiguration) protected readonly serviceConfig: ServiceConfiguration,
+        @inject(Logger) private readonly logger: Logger,
+    ) {
+        super();
+    }
+
+    public async handleRequest(): Promise<void> {
+        const reportId = <string>this.context.bindingData.reportId;
+        if (!this.guidGenerator.isValidV6Guid(reportId)) {
+            this.context.res = {
+                status: 422,
+                body: `Unprocessable Entity: ${reportId}.`,
```

Comment:
Change to "invalid report id" or something.

Reply:
done.